src/components/_global/icons/ArrowDownIcon.vue

```diff
@@ -0,0 +1,14 @@
+<template>
+  <svg
```

Comment:
Can we use the feather icons compared to this? or are we moving away from feather @garethfuller

Reply:
I took the one Pon used in his designs - it's more "pointy" than the feathers one... I do think we should stick to whatever Pon uses in his designs.